index.bs

```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft

```

Comment:
s/ Draft//

Reply:
(It's unnecessary to say that it's a draft in the title, as the Status metadata item establishes that.)